cocos/2d/renderer/batcher-2d.ts

```diff
@@ -225,6 +225,11 @@ export class Batcher2D implements IBatcher {
     }
 
     public uploadBuffers () {
+        this._meshDataArray.forEach(rd => {
```

Comment:
This should also be wrapped into the following if check, if no batches, then no need to upload any buffer